templates/scenarios/js/notification-restify/teamsfx/app.yml

```diff
@@ -1,12 +1,12 @@
 version: 1.0.0
 
 provision:
-  - uses: botAadApp/create # Creates a new AAD app for Azure Bot service if BOT_ID environment variable is empty
+  - uses: botAadApp/create # Creates a new AAD app for Bot Registration.
```

Comment:
We cannot just say `for Azure Bot service` here because this driver is also used in local debug.